src/__integration_tests__/paymentflow.integration.test.ts

```diff
@@ -198,6 +200,30 @@ describe("Checkout payment activation failure tests", () => {
     expect(resultMessage).toContain("PPT_PSP_SCONOSCIUTO");
   });
 
+  it("Should fail a satispay payment ACTIVATION and get PPT_WISP_SESSIONE_SCONOSCIUTA", async () => {
+    /*
+     * Satispay payment with notice code that fails on activation and get PPT_WISP_SESSIONE_SCONOSCIUTA
+     * and redirect to expired session page
+     */
+
+    await fillAndSubmitSatispayPayment(FAIL_ACTIVATE_502_PPT_WISP_SESSIONE_SCONOSCIUTA, VALID_FISCAL_CODE, EMAIL);
+    await page.waitForSelector("#sessionExpiredMessageTitle")
+
+    expect(page.url()).toContain("/sessione-scaduta");
```

Comment:
add assertions against displayed messages too (it language test should be sufficient)

Reply:
Done here 9c136f7fa8bcd6b721060e98e0270023f40a560a